app/src/main/java/org/fossasia/openevent/utils/SpeakerIntent.java

```diff
@@ -0,0 +1,64 @@
+package org.fossasia.openevent.utils;
+
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.Toast;
+
+import org.fossasia.openevent.R;
+import org.fossasia.openevent.data.Speaker;
+
+/**
+ * Created by MananWason on 02-07-2015.
+ */
+public class SpeakerIntent {
+    Speaker speaker;
+    //    Intent intent;
+    Context context;
```

Comment:
Yeah I was not able to get context at that time, so I cached it. But I have removed it now.